### PR TITLE
Downgrade absl version.

### PR DESCRIPTION
**Pull request by ggreenway**
The current version is causing ASAN build failures in some environments.

See https://github.com/abseil/abseil-cpp/issues/118

Signed-off-by: Greg Greenway <ggreenway@apple.com>

*Risk Level*: Low

*Testing*: Tests pass

### CODE REVIEW

**Comment by ggreenway** [comment]
cc @jmarantz

File: bazel/repository_locations.bzl [hunk 1]
```diff
@@ -5,7 +5,7 @@ REPOSITORY_LOCATIONS = dict(
         remote = "https://github.com/google/boringssl",
     ),
     com_google_absl = dict(
-        commit = "bd5a2bba8f4238db2e06c99ac808fc8327dec2a0",  # 2018-05-07
+        commit = "26b789f9a53d086c8b8c9c2668efb251e37861cd",  # 2018-05-04
```
Comment (jmarantz) [inline bazel/repository_locations.bzl]: Is there an upstream bug for abseil? We should probably reference it in the PR and here in a comment, to avoid having the next person come in here and inadvertently revert the change.

Comment (ggreenway) [reply bazel/repository_locations.bzl]: Upstream bug is referenced in the commit message. I'll add a comment in this file as well.
